jumpscale/clients/stellar/stellar.py

```diff
@@ -892,6 +911,8 @@ def manage_sell_order(
                 raise RuntimeError(f"couldn't sumbit transaction, probably unfunded") from e
```

Comment:
typo should be submit instead of sumbit

Reply:
actually, this whole message should be refactored, cause it only shows underfunded transaction, we can occur more errors like offer not found or whatever